doc/whats_new/v0.20.rst

```diff
@@ -152,6 +152,11 @@ Preprocessing
   missing values. :issue:`8075` by :user:`Maniteja Nandana <maniteja123>` and
   :user:`Guillaume Lemaitre <glemaitre>`.
 
+- Added :class:`impute.ChainedImputer`, which is a strategy for imputing missing
```

Comment:
This shouldn't be here

Reply:
I'm pretty sure I did not add that. Comes from the merge.

Reply:
Still, please remove this, it shouldn't be in the diff.